services/dy-dash/cc-rabbit-0d/HOWTOBAKE.md

```diff
@@ -0,0 +1,45 @@
+# cc-rabbit-0d - How to Bake your Cookie
+
```

Comment:
This file is not needed anymore after the cookie is applied. Would move the instructions to the main readme perhaps